.editorconfig

```diff
@@ -14,17 +14,5 @@ indent_size = 4
 indent_style = tab
 
 [*.d]
-dfmt_align_switch_statements = true
-dfmt_brace_style = allman
-dfmt_outdent_attributes = true
-dfmt_outdent_labels = true
-dfmt_soft_max_line_length = 80
-dfmt_space_after_cast = true
-dfmt_space_after_keywords = true
-dfmt_split_operator_at_line_end = false
-end_of_line = lf
-indent_size = 4
-indent_style = space
-insert_final_newline = true
-max_line_length = 120
-tab_width = 8
+dfmt_single_template_constraint_indent = true
```

Comment:
Why are these changes here? Intentional?

Reply:
Yes, the configs that were removed were redundant, because they're the defaults. The two lines that were added were to maintain consistency between multi-line formatting across the codebase (the default behaviour uses two tabs to indent, which is somewhat unidiomatic, imo). There's also a [discussion on the forum](https://forum.dlang.org/subscription-posts/lhugkccytdyaoybpczww) about whether to keep this knob or not on dmdfmt.

Reply:
Its good practice (for good reason) to make PRs as small as possible. If you want to change this it should be a separate pull request for that reason.